plugins/carlabase/carla.cpp

```diff
@@ -495,10 +570,36 @@ CarlaInstrumentView::CarlaInstrumentView(CarlaInstrument* const instrument, QWid
     m_toggleUIButton->setWhatsThis(
                 tr( "Click here to show or hide the graphical user interface (GUI) of Carla." ) );
 
-    l->addWidget( m_toggleUIButton );
-    l->addStretch();
-
+	// Refresh params button
+	m_refreshParamsButton = new QPushButton( tr( "" ), this );
+	m_refreshParamsButton->setIcon( embed::getIconPixmap( "reload" ) );
+	m_refreshParamsButton->setWhatsThis(
+                tr( "Click here to reload the Carla parameter knobs." ) );
+    QSizePolicy sizePolicy(QSizePolicy::Maximum, QSizePolicy::Fixed);
+	sizePolicy.setHorizontalStretch(0);
+	sizePolicy.setVerticalStretch(0);
+	sizePolicy.setHeightForWidth(m_refreshParamsButton->sizePolicy().hasHeightForWidth());
+	m_refreshParamsButton->setSizePolicy(sizePolicy);
+	m_refreshParamsButton->setCheckable( false );
```

Comment:
Mixing old and new coding styles doesn't look good. I suggest putting whitespaces either anywhere or nowhere.